packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -29,6 +29,10 @@ export 'package:flutter/services.dart' show TextSelectionDelegate;
 /// called.
 const Duration _kDragSelectionUpdateThrottle = Duration(milliseconds: 50);
 
+/// A duration that determines the delay before showing the text selection
+/// toolbar again after dragging either selection handle on Android.
+const Duration _kAndroidPostDragShowDelay = Duration(milliseconds: 300);
```

Comment:
Can you add a comment like this about where you got this value? "Eyeballed on a physical Android <version number> <model e.g. Pixel 6 etc.> device."

That way we know if/when/how we need to update it if someone notices it doesn't match.